api/tests/integration/infrastructure/repositories/clea-certified-candidate-repository_test.js

```diff
@@ -0,0 +1,215 @@
+const CleaCertifiedCandidate = require('../../../../lib/domain/models/CleaCertifiedCandidate');
+const { PIX_EMPLOI_CLEA_V3 } = require('../../../../lib/domain/models/Badge').keys;
+
+describe('Integration | Repository | clea-certified-candidate-repository', function () {
+  describe('#getBySessionId', function () {
+    describe('when there is several candidates for clea certification in the session', function () {
+      it('returns the list of clea certified candidates only', async function () {
+        // given
+        const userId = databaseBuilder.factory.buildUser({ email: 'jean-mi@coco.fr' }).id;
+        const userId2 = databaseBuilder.factory.buildUser({ email: 'marie-mi@coco.fr' }).id;
+        const candidateCleaSuccess = {
+          firstName: 'Jean-Mi',
+          lastName: 'Mi',
+          birthdate: '2001-02-07',
+          birthplace: 'Paris',
+          sex: 'M',
+          birthPostalCode: null,
+          birthCountry: 'FRANCE',
+          createdAt: new Date('2020-02-01'),
+          userId,
+        };
+        const candidateCleaSuccess2 = {
+          firstName: 'Marie',
+          lastName: 'Ri',
+          birthdate: '2001-02-04',
+          birthplace: 'Orléans',
+          sex: 'F',
+          birthPostalCode: null,
+          birthCountry: 'FRANCE',
+          createdAt: new Date('2020-02-01'),
+          userId: userId2,
+        };
+        const candidateWithoutClea = {
+          firstName: 'Jean-Michel',
+          lastName: 'Chel',
+        };
+
+        const sessionId = databaseBuilder.factory.buildSession().id;
+        const badgeClea = databaseBuilder.factory.buildBadge({ id: 1, isCertifiable: true, key: PIX_EMPLOI_CLEA_V3 });
+
+        databaseBuilder.factory.buildCertificationCourse({ id: 91, sessionId, ...candidateCleaSuccess });
+        databaseBuilder.factory.buildCertificationCourse({ id: 92, sessionId, ...candidateCleaSuccess2 });
+        databaseBuilder.factory.buildCertificationCourse({ id: 93, sessionId, ...candidateWithoutClea });
+
+        databaseBuilder.factory.buildComplementaryCertification({
+          id: 1,
+          key: CLEA,
+        });
+
+        databaseBuilder.factory.buildComplementaryCertificationCourse({
+          id: 991,
+          certificationCourseId: 91,
+          complementaryCertificationId: 1,
+          complementaryCertificationBadgeId: databaseBuilder.factory.buildComplementaryCertificationBadge({
+            complementaryCertificationId: 1,
+            badgeId: badgeClea.id,
+          }).id,
+        });
+        databaseBuilder.factory.buildComplementaryCertificationCourse({
+          id: 992,
+          certificationCourseId: 92,
+          complementaryCertificationId: 1,
+          complementaryCertificationBadgeId: databaseBuilder.factory.buildComplementaryCertificationBadge({
+            complementaryCertificationId: 1,
+            badgeId: badgeClea.id,
+          }).id,
+        });
```

Comment:
Le complementaryCertificationBadge n'est pas lié à une certification specifique. Il sera identique pour les 2:
```suggestion
       const complementaryCertificationBadgeId: databaseBuilder.factory.buildComplementaryCertificationBadge({
            complementaryCertificationId: 1,
            badgeId: badgeClea.id,
          }).id,
        databaseBuilder.factory.buildComplementaryCertificationCourse({
          id: 991,
          certificationCourseId: 91,
          complementaryCertificationId: 1,
          complementaryCertificationBadgeId,
        });
        databaseBuilder.factory.buildComplementaryCertificationCourse({
          id: 992,
          certificationCourseId: 92,
          complementaryCertificationId: 1,
          complementaryCertificationBadgeId,
        });
```